assets/panel/debugger.properties

```diff
@@ -539,15 +547,27 @@ sourceSearch.noResults2=No results found
 
 # LOCALIZATION NOTE (ignoreExceptions): The pause on exceptions button tooltip
 # when the debugger will not pause on exceptions.
-ignoreExceptions=Ignore exceptions. Click to pause on uncaught exceptions
+ignoreExceptionsTooltip=Ignore exceptions. Click to pause on uncaught exceptions
+
+# LOCALIZATION NOTE (ignoreExceptions): The pause on exceptions button description
+# when the debugger will not pause on exceptions.
+ignoreExceptions=Ignore exceptions
```

Comment:
Is the case (sentence case) correct for a button? Same question for other strings used as button labels later in the PR